data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
+				decline
+	to complete
+		or
+			has "Remnant: Heavy Laser: done"
+			has "Remnant: Catalytic Ramscoop: done"
+			has "Remnant: Plasma Cannons: done"
+			has "Remnant: Electron Beam: done"
+			has "Remnant: D94-YV Shield Generator: done"
+			has "Remnant: S-970 Regenerator: done"
+
+
+
 mission "Remnant: Heavy Laser"
 	name "Retrieve Heavy Lasers for the Remnant"
-	description "A Remnant engineer on <planet> has offered to pay you <payment> in exchange for obtaining two heavy lasers for them to study."
+	description "A Remnant engineer has offered to pay you <payment> in return for delivering two heavy lasers to <planet> for them to study."
 	minor
-	source
+	source "Viminal"
+	destination
 		government "Remnant"
 	to offer
-		has "Remnant: Technology Available: offered"
-		not "Remnant: Catalytic Ramscoop: active"
-		not "Remnant: Catalytic Ramscoop: done"
-		not "Remnant: Plasma Cannons: active"
-		not "Remnant: Plasma Cannons: done"
-		random < 50
+		or
+			has "Remnant: Tech Retrieval: active"
+			has "Remnant: Tech Retrieval: done"
 	on offer
 		require "Heavy Laser"
 		conversation
-			`As you are browsing the spaceport shops, a woman approaches you and introduces herself as a researcher at one of the Remnant weapons facilities. She awkwardly pauses before beginning to recite a quick tune: "We can see that your ship has beam weapons that match human techniques, but are unlike anything we have record of. What kind of weapons are they?"`
+			`You remember that Taely was interested in seeing examples of technology from human space. Would you like to show her your heavy laser cannons?`
 			choice
-				`	"They're heavy lasers."`
-				`	"I'd rather not share that."`
-					decline
-			
-			`	"Really? Our records indicate that humanity had primitive laser technology at the time of the Exodus, but nothing that would be considered useful as weapons." She pauses to look down at her scanner. "And yet our scans indicate that they are capable of significant power output.`
-			`	"Based on these scans, we may well be able to learn quite a bit from these weapons. Would you be able to acquire a pair of them for us? We could offer you very good compensation for doing so."`
-			choice
-				`	"Sure, I would be glad to accept that deal."`
-				`	"Sorry, I'm not really interested in helping Remnant research."`
-					decline
-
-			`	After a bit of haggling, she agrees to pay you <payment> in exchange for two heavy lasers.`
+				`	Not now.`
+					defer
+				`	Yes.`
+			branch stuff
+				has "Remnant Tech Retrieval: done"
+			`	After asking a few people, you finally get directed to a bay in the shipyard where you find Taely just finishing the installation of a thrasher cannon in a starling. "Greetings, <first>. I'll be down in a minute." She does a few more things inside a panel, reseals the hatch, and slides down a fin to land next to you. "So, do you have something for me?"`
+				goto the_rest
```

Comment:
~~Have we taken steps backwards? Missing tabs within the backticks? Two work labels again?~~ You could probably use "next" or "continue" as a label here. Same with the rest of these missions.

Reply:
Changed to "next".